packages/shared/src/components/notifications/EnableNotification.tsx

```diff
@@ -30,10 +29,17 @@ type EnableNotificationProps = {
   parentCommentAuthorName?: string;
 };
 
+const containerClassName: Record<NotificationPromptSource, string> = {
+  notificationList: 'px-6 w-full border-l border-theme-color-cabbage',
```

Comment:
It seems the border-theme-color-cabbage can actually be set on the main attribute as it's recurring for all 3 versions.